### PR TITLE
Don't warning build non Kprobes

### PR DESCRIPTION
**Pull request by diphons**
KernelSU work without Kprobes
Tested on kernel xiaomi linux 4.19

### CODE REVIEW

File: kernel/ksu.c [hunk 1]
```diff
@@ -56,7 +58,7 @@ int __init kernelsu_init(void)
 #ifdef CONFIG_KPROBES
 	ksu_enable_sucompat();
 	ksu_enable_ksud();
-#else
+#elif LINUX_VERSION_CODE < KERNEL_VERSION(4, 19, 0)
```
Comment (tiann) [inline kernel/ksu.c:61]: Can we remove the condition, and make the `warning` a `info`?

**Review by tiann — CHANGES_REQUESTED** [review]

File: kernel/ksu.c (hunk 1 above)
Comment (diphons) [reply kernel/ksu.c:61]: Ya bro.. maybe that's better..
I tried on linux 4.9.x without kprobes, and KSU runs normally too

Comment (tiann) [reply kernel/ksu.c:61]: 4.9 and 4.14 should also work without kprobes, so we shouldn't warn them, too.

Comment (diphons) [reply kernel/ksu.c:61]: Ya bro...

**Comment by tiann** [comment]
I will close this since it is stale, welcome to make another PR in any time!

**Pull request closed without merging**